app/page/info/activity.tsx

```diff
@@ -36,30 +45,201 @@ const UserOpHistoryItem: React.FC<{
   chainName: string
 }> = ({ txLog, chainName }) => {
   const { createdAt, status } = txLog
-  const creationDate = new Date(createdAt * 1000).toLocaleDateString()
+  const creationDate = new Date(createdAt * 1000).toLocaleDateString("zh-TW", {
+    year: "numeric",
+    month: "numeric",
+    day: "numeric"
+  })
+  const creationTime = new Date(createdAt * 1000).toLocaleTimeString("zh-TW", {
+    hour: "numeric",
+    minute: "numeric",
+    second: "numeric",
+    hour12: false
+  })
 
-  if (
-    txLog.status === TransactionStatus.Succeeded ||
-    txLog.status === TransactionStatus.Failed
-  ) {
+  if (txLog.status === TransactionStatus.Succeeded) {
     const userOpHash = txLog.receipt.userOpHash
     return (
-      <div>
-        <span>{`${creationDate}: `}</span>
-        <span>{`${status} `}</span>
-        <a
-          href={`${explorerUrl}userOpHash/${userOpHash}?network=${chainName}`}
-          target="_blank">
-          {`${address.ellipsize(userOpHash)}`}
-        </a>
+      <a
+        className="w-full flex items-center p-[14px_0px_14px_0px]"
+        href={`${explorerUrl}userOpHash/${userOpHash}?network=${chainName}`}
+        target="_blank">
+        {/* Status and to address */}
+        <div className="flex-grow flex flex-col items-start">
+          {/* Status and token amount */}
+          <div className="flex items-center mb-[4px]">
+            <ArrowUpRight className="w-[16px] h-[16px] mr-[8px]" />
+            <div className="mr-[8px] leading-[19px] text-[16px] font-[Inter] font-[400] text-[#000000]">
+              Send
+            </div>
+            <div className="mr-[8px] leading-[19px] text-[16px] font-[Inter] font-[400] text-[#000000]">
+              1.2
+            </div>
+            <div className="leading-[19px] text-[16px] font-[Inter] font-[400] text-[#000000]">
+              ETH
+            </div>
+          </div>
+          {/* To address */}
+          <div className="leading-[19px] text-[16px] font-[Inter] font-[400] text-[#bbbbbb] whitespace-nowrap">
+            to: 0x0000...00000
+          </div>
+        </div>
+        {/* Transaction time */}
+        <div>
+          <div className="flex flex-col items-end">
+            <div className="mb-[4px] leading-[19px] text-[16px] font-[Inter] font-[400] text-[#000000]">
+              {creationDate}
+            </div>
+            <div className="leading-[19px] text-[16px] font-[Inter] font-[400] text-[#000000]">
+              {creationTime}
+            </div>
+          </div>
+        </div>
+      </a>
+    )
+  }
```

Comment:
maybe we could extract these activity entries into a reuseful component since they're quite alike.

Reply:
The Activity compont is temporary, it will be updated in the next PR to match the latest Figma design.